external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/trident/KafkaTridentSpoutBatchMetadata.java

```diff
@@ -74,8 +74,8 @@ public TopicPartition getTopicPartition() {
 
     @Override
     public String toString() {
-        return "KafkaTridentSpoutBatchMetadata{" +
-                "topicPartition=" + topicPartition +
+        return super.toString() +
```

Comment:
All this is doing is that the method `toString()` instead of printing `KafkaTridentSpoutBatchMetadata` it will print `org.apache.storm.kafka.spout.trident.KafkaTridentSpoutBatchMetadata@6b74f986`. This was useful while I was debugging and writing the code because the code is multi threaded for multiple executors, and I needed to see which object instance (e.g `@6b74f986`) was being passed around. I can revert it back, but for me there was benefit in having it like this.